charts/hivemq-platform/values.yaml

```diff
@@ -88,17 +88,41 @@ restApi:
   # Enables or disables authentication and authorization.
   authEnabled: false
 
-# Configures the HiveMQ License information.
+# Configures the HiveMQ Platform license information.
```

Comment:
```suggestion
# Configures the HiveMQ license information.
```

Reply:
In this particular case, I changed it like this cause this whole `license` section includes all the different licenses for the entire platform. Only `HiveMQ license` would be misleading IMO - Maybe something like below, what do you think?

```yaml
Configures all HiveMQ licenses information.
```